js/autofillSetup.js

```diff
@@ -10,13 +10,12 @@ const AutofillSetup = {
     }
 
     manager.checkIfConfigured().then((configured) => {
-      return { manager, configured }
-    }).then((result) => {
-      const { manager, configured } = result
-      if (manager && !configured) {
-        if (manager.name === 'Bitwarden') {
-          showBitwardenDialog()
-        }
+      let setupMethod = 'dragdrop'
+      if (platformType === 'mac' && manager.name === '1Password') {
+        setupMethod = 'installer'
+      }
```

Comment:
since `setupMethod` is determined by the manager type, I think it's better to move this logic to the manager classes, so they can tell us how to setup them. something like `manager.getSetupMethod(platformType)`